fix_includes_test.py

```diff
@@ -107,10 +107,12 @@ def RegisterFileContents(self, file_contents_map):
     for (filename, contents) in file_contents_map.items():
       before_contents = []
       expected_after_contents = []
-      for line in contents.splitlines():
+      for line in contents.splitlines(True):
         m = remove_re.search(line)
         if m:
-          before_contents.append(line[:m.start()])
+          # The trailing line seperator is stripped
```

Comment:
Typo: seperator -> separator.

Collapse this to a single sentence, including closing period:

    # The trailing line separator is stripped, so append a '\n'.

Reply:
Fixed in latest